stixcore/processing/publish.py

```diff
@@ -539,12 +539,12 @@ def publish_fits_to_esa(args):
 
     parser.add_argument("-l", "--include_levels",
                         help="what levels should be published", type=str,
-                        default=CONFIG.get('Publish', 'include_levels', fallback="L0, L1, L2"))
+                        default=CONFIG.get('Publish', 'include_levels', fallback="L0, L1, L2, ANC"))
 
     parser.add_argument("-p", "--include_products",
                         help="what products should be published", type=str,
                         default=CONFIG.get('Publish', 'include_products',
-                                           fallback="ql,hk,sci,aux,cal"))
+                                           fallback="ql,hk,sci,asp,cal"))
```

Comment:
asp = aspect or should it be anc = ancialliar?

Reply:
yes: asp = aspect 
ANC is allready in the level
and i added a new type as it is not HK anymore